daemons/fenced/pacemaker-fenced.c

```diff
@@ -123,10 +123,8 @@ st_ipc_dispatch(qb_ipcs_connection_t * qbc, void *data, size_t size)
     if (c->name == NULL) {
         const char *value = crm_element_value(request, F_STONITH_CLIENTNAME);
 
-        if (value == NULL) {
-            value = "unknown";
-        }
-        c->name = crm_strdup_printf("%s.%u", value, c->pid);
+        c->name = crm_strdup_printf("%s.%u",
+                                    pcmk__s(value, PCMK_VALUE_UNKNOWN), c->pid);
```

Comment:
This is arbitrary text rather than defined values, but no big deal

Reply:
Thanks, yeah this sn't a good fit. I got into the habit of using `PCMK_VALUE` with almost anything that's used with `crm_xml_add()`/`crm_element_value()`. If `value != NULL` here, then it comes from `crm_element_value()`.

But the client name itself is arbitrary and can even include the PID.